reports/TC39/2017-01.md

```diff
@@ -0,0 +1,124 @@
+
+### 64 Bit Integers / BigInts
+
+I will not go deeply into [this proposal](https://gist.github.com/BrendanEich/4294d5c212a6d2254703), but I want to note that for OS interoperability purposes,
+the Node.js team has a strong need for 64 bit integers. In addition to these use cases, there are significant usages for 64 bit integers in cryptography and gaming.
+
+There is a feeling on the committee that it may be better to implement a bigint type - that is to say a type that is not limited to 64 bits, and can be optimized
+to the appropriate size based on the code, variables, and underlying system as appropriate. However, this presents more work in specification. If anybody has specific
+arguments about why a bigint type would be strongly preferable to an int64 type, we would be interested to hear your argument.
+
+This proposal did not advance, due to need for feedback.
+
+### IEEE 754 Sign Bit
+
+If you have ever dealt with ECMAScripts famous `-0` then you are familiar with the inconvenience not having a way to explicitly ask for the sign of an IEEE 754
+floating point number is. For those who aren't familiar with this issue, observe the following:
+
+```js
+Math.sign(2) // 1
+Math.sign(-2) // -1
+Math.sign(0) // 0
+Math.sign(-0) // -0
+```
+
+This is not terribly helpful when one wants to determine the actual sign of a number while handling all edge cases. [This proposal](http://jfbastien.github.io/papers/Math.signbit.html) introduces a new
+api, `Math.signbit()` that resolves this issue. Observe:
+
+```js
+Math.signbit(2) //false
+Math.signbit(-2) //true
+Math.signbit(0) //false
+Math.signbit(-0) //true
+```
+
+As you can see, asking if -0 is negative will in fact result in -0 being negative. WIN!
+
+This proposal advanced to stage 1 at this meeting.
+
+### Summary
+
+There were many other proposals discussed at the meeting, and I encourage you to look at everything that is in process on the [TC39 GitHub repository](https://github.com/tc39/proposals).
+
+## Community!
+
+Right now, TC39 is in the process of regrouping on the ECMAScript vision for the future. As part of this, several committee members will be reaching out to local
+user groups to hold discussions with users about what they want from the future of their programming language. I would love to hear more from any interested parties
+about things you hope to see. Feel free to reach out to me on twitter (@maggiepint) with any questions, or to [raise an issue on the JS Foundation standards repository](https://github.com/JSFoundation/standards).
+
+In addition, if you want to contribute your opinions, remember that TC39 discusses proposals on [GitHub](https://github.com/tc39) where you are welcome to have a voice.
+
+Finally, you can contribute code! TC39 maintains a test suite called [test262](https://github.com/tc39/test262) that validates conformance of implementations to the spec. They would love more help to get further test
```

Comment:
s/validates conformance of implementations to the spec/provides specs features acceptance/

The specs are just a recommendation, with it, TC39 does not require specific conformance. The difference is subtle.